plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -0,0 +1,270 @@
+Switch the user role to postgres and start postgres client
+```
+  su - postgres -c psql
+```
+
+Once inside the client, we need to create two databases and dedicated roles, one for foreman and one for candlepin
+```
+  CREATE USER "foreman" WITH PASSWORD '<FOREMAN_PASSWORD>';
+  CREATE USER "candlepin" WITH PASSWORD '<CANDLEPIN_PASSWORD>';
+  CREATE DATABASE foreman OWNER foreman;
+  CREATE DATABASE candlepin OWNER candlepin;
+```
+
+### Test it works
+From `katello.example.com` test the DB is accessible:
+```
+PGPASSWORD='<FOREMAN_PASSWORD>' psql -h postgres.example.com  -p 5432 -U foreman -d foreman -c "SELECT 1 as ping"
+```
+```
+PGPASSWORD='<CANDLEPIN_PASSWORD>' psql -h postgres.example.com  -p 5432 -U candlepin -d candlepin -c "SELECT 1 as ping"
+```
+If there are no errors we are done with database preparation. 
+
+## Prepare remote Mongo
+GOAL: To use remote Mongo database with Katello we have to:
+* be able to access the databases from katello box
+* the database user we use to connect to the database needs to own the database
+
+### Install Mongo DB
+Warning: This is just minimal testing setup which is not suitable for production.
+
+Assume our Mongo server has hostname `mongo.example.com`.
+Install and enable Mongo server
+``` 
+  yum -y localinstall https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  yum install -y mongodb mongodb-server
+```
+Enable authentication in `/etc/mongod.conf`
+```
+auth=true
+```
+
+Enable and start the service
+```
+  systemctl start mongod
+  systemctl enable mongod
+```
+
+### Create Pulp user and database 
+```
+  mongo admin -u admin -p admin --eval "db.createUser({user:'pulp',pwd:'<PULP_PASSWORD>',roles:[{role:'dbOwner', db:'pulp_database'},{ role: 'readWrite', db: 'pulp_database'}]})"
+
+```
+
+### Test it works
+From `katello.example.com` test the mongo DB is accessible:
+```
+  mongo --host mongo.example.com -u pulp -p <PULP_PASSWORD> --port 27017 --eval 'ping:1' pulp_database
+```
+If there are no errors we are done with database preparation. 
+
+## Fresh install
+
+### Install katello package
+We assume the box where the Katello server will be installed has hostname `katello.example.com`.
+
+Follow the [documentation](plugins/katello/{{ page.version }}/installation/index.html) to install the `katello` package and **do not run foreman-installer.** We need to use the remote database flags with the installer. Use the following steps once the katello rpm is installed.
+
+
+### Prepare remote databases
+Follow the instructions to [prepare remote mongo](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-mongo) and [prepare remote postgres](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-postgres) to make the remote database servers ready for installation.
+
+### Run the installer
+To install and configure Katello we just need to run 
+```
+  foreman-installer --scenario katello \
```

Comment:
You probably also want `--foreman-db-manage false` so it doesn't install a local postgresql server. The same goes for `--katello-pulp-manage-db false`.